src/routes/(admin)/questionbank/import/+page.svelte

```
@@ -109,7 +108,7 @@
 					Use our template to ensure your data is formatted correctly for upload.
 				</p>
 			</div>
-			<a href={BulkTemplate} download="template.csv">
+			<a href="/api/questions/bulk-template" download="template.csv">
```

ESLint

[error] Unexpected href link without resolve().

(svelte/no-navigation-without-resolve)